### PR TITLE
move docs into monorepo

### PR DESCRIPTION
**Pull request by mwcz**
Move documentation into the main repo.  The two major benefits of this are:

Pull requests can contain both code and related docs, together at last.  Docs can even be required before approving a PR, similar to how tests are/should be required.

With docs as a first-class member of the main repo, everyone will have closer proximity to them and a better understanding of how to update; it's currently an oft-forgotten repo that few people use.

### CODE REVIEW

File: scripts/dev.sh [hunk 1]
```diff
@@ -6,4 +6,13 @@ for el in "$@"; do
   CMD="$CMD --scope \"*/$el\""
 done
 
+source scripts/hugo-check.sh
+if hugoCheck; then
+  pushd docs > /dev/null
+  hugo server &
+  popd > /dev/null
+else
+  echo "Not running Hugo server (docs site) becuase hugo is not installed."
```
Comment (castastrophe) [inline scripts/dev.sh]: minor typo: because

**Comment by castastrophe** [comment]
`npm run start` failed with the following:

```
patternfly-elements  (docs-docs-docs) $ npm run start

> @patternfly/patternfly-elements@ start /Users/carobert/repos/patternfly-elements
> npm run doc-listing-inject && spandx


> @patternfly/patternfly-elements@ doc-listing-inject /Users/carobert/repos/patternfly-elements
> node scripts/doc-listing-inject.js

/Users/carobert/repos/patternfly-elements/scripts/doc-listing-inject.js:29
  if (err) throw err;
           ^

Error: ENOENT: no such file or directory, open '/Users/carobert/repos/patternfly-elements/doc/wrapper.html'
```

**Comment by mwcz** [comment]
Thanks, fixed!

File: scripts/deploy-docs.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+source hugo-check.sh
```
Comment (castastrophe) [inline scripts/deploy-docs.sh]: Would it be potentially easier to set up hugo to install with npm using https://www.npmjs.com/package/hugo-installer?

Comment (castastrophe) [reply scripts/deploy-docs.sh]: There's also a hugo-cli that might be useful https://www.npmjs.com/package/hugo-cli

**Comment by mwcz** [comment]
Yes that is awesome!!!

File: scripts/deploy-docs.sh (hunk 2 above)
Comment (mwcz) [reply scripts/deploy-docs.sh]: Yes that is awesome!!!

Comment (mwcz) [reply scripts/deploy-docs.sh]: I won't have time to add those packages this sprint, so I made #352 to keep track of it.  I'd like to merge this PR as-is and add those next sprint if no one objects.

File: scripts/dev.sh (hunk 1 above)
Comment (mwcz) [reply scripts/dev.sh]: Thanks, fixed!

File: scripts/deploy-docs.sh (hunk 2 above)
Comment (castastrophe) [reply scripts/deploy-docs.sh]: 👍

**Comment by castastrophe** [comment]
Just for posterity: confirming that I tested `npm run start`, `npm run build`, `npm run storybook`, and `npm run dev` to ensure no regressions.

**Pull request merged**